src/masternode-payments.cpp

```diff
@@ -1115,8 +1115,11 @@ void AdjustMasternodePayment(CMutableTransaction &tx, const CTxOut &txoutMastern
         if(tposContract.IsValid()) // here we have 3 outputs, first as stake reward, second as tpos reward, third as MN reward
         {
             masternodePayment /= 100; // to calculate percentage
-            tx.vout[i - 1].nValue -= masternodePayment * tposContract.stakePercentage; // adjust reward for owner.
-            tx.vout[i].nValue -= masternodePayment * (100 - tposContract.stakePercentage); // adjust reward for merchant
+            tx.vout[i - 1].nValue -= masternodePayment * (100 - tposContract.nOperatorReward); // adjust reward for owner.
```

Comment:
I'd create a common function for this, and make sure it has proper tests, a mistake would be very bad for us

Reply:
true